Source/UI/NewUI/PaymentForm/Factory/PaymentFormFactory.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+public struct PaymentFormFactory {
+
+    public static func getBillingFormButtonView(style: CellButtonStyle, delegate: SelectionButtonViewDelegate?) -> (view: UIView, delegate: SelectionButtonViewDelegate?)  {
+        let view = SelectionButtonView()
+        view.delegate = delegate
+        view.update(style: style)
+        return (view, view.delegate)
```

Comment:
We may have discussed this previously, but why is there a delegate being returned?